src/components/Profile/components/MembershipsList/index.jsx

```diff
@@ -66,14 +70,12 @@ const MembershipsList = ({ username, myProf, createSnackbar }) => {
   );
   const noteInfo = myProf && (
     <div align="left" className={styles.memberships_card_note}>
-      <Typography>NOTE:</Typography>
-      <ul>
-        <li>
-          <Typography>
-            Shaded areas are visible only to you and other members of the same club session.
-          </Typography>
-        </li>
-      </ul>
+      <Typography>
```

Comment:
Since there is no styling or links inside this note I think we can dispense with the {' '} and the <span> ... </span> tags.